recipe/deploy/vendors.php

```diff
@@ -28,5 +46,9 @@
     if (!commandExist('unzip')) {
         warning('To speed up composer installation setup "unzip" command with PHP zip extension.');
     }
-    run('cd {{release_or_current_path}} && {{bin/composer}} {{composer_action}} {{composer_options}} 2>&1');
+    $paths = get('vendor_paths');
+    foreach ($paths as $path) {
+        $commands[] = "cd $path && {{bin/composer}} {{composer_action}} {{composer_options}} 2>&1";
+    }
+    run(implode('; ', $commands));
```

Comment:
Just run `run()` inside the loop, you save a variable and iteration then :)

Reply:
Taken from the `clear_paths` recipe:
https://github.com/deployphp/deployer/blob/cf569530f5f030177b324a2d276b6df62dfbba87/recipe/deploy/clear_paths.php#L20-L23